example/gen/haberdasher_twirp.rb

```diff
@@ -7,4 +7,10 @@ class HaberdasherService < Twirp::Service
     service "Haberdasher"
     rpc :HelloWorld, HelloWorldRequest, HelloWorldResponse, :handler_method => :hello_world
   end
+
+  class HaberdasherProtoClient < Twirp::ProtoClient
+    package "example"
+    service "Haberdasher"
+    rpc :HelloWorld, HelloWorldRequest, HelloWorldResponse, :handler_method => :hello_world
```

Comment:
Since the client doesn't have a handler, the option `:handler_method` should be removed. If we would prefer clients to have methods like `client.hello_world(...)` instead of `client.rpc(:HelloWorld, ...)`, then an option like `:client_method` would make more sense.